Extensions/BitBucket/Src/vss-extension.json

```diff
@@ -194,6 +194,33 @@
                   "dataType": "string",
                   "maxLength": 300
               }
+          },
+          {
+            "id": "defaultVersionType",
+            "name": "i18n:Default version",
```

Comment:
we don't support localization for strings in extensions in this repo. so don't think we need i18n prefix

Reply:
removed the i18n prefixes